src/one_dragon_qt/view/standalone_app_run_interface.py

```
@@ -199,10 +200,21 @@ def __init__(self, available_apps: dict[str, str], parent: QWidget):
         item_font = getFont(16)
+        items = []
         for _, app_name in available_apps.items():
             item = QListWidgetItem(gt(app_name))
             item.setSizeHint(QSize(0, 36))
             item.setFont(item_font)
+            items.append(item)
+
+        # 按拼音排序以便查找
+        try:
+            locale.setlocale(locale.LC_COLLATE, "zh_CN.UTF-8")
+        except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)